### PR TITLE
Remove whitespace, ProUI fixes

### PR DESCRIPTION
**Pull request by classicrocker883**
### Description
removes whitespace in several files

fixes the Icons in popups to be centered in ProUI.

park when print aborts in ProUI.

fix backlight timeout setting minutes would change itself to something odd like 256 whenever checkbox sound enabled/disabled.

### Benefits
removing whitespace to be consistent with the rest of code.

looks aesthetically better in ProUI to center icons based on their own dimensions.
have it move to a park position when print is aborted, and then disable stepper motors (if so defined)

### Related Issues

recent commit fixes/corrects setting backlight timeout.
https://github.com/classicrocker883/MRiscoCProUI/issues/77

### CODE REVIEW

File: Marlin/src/lcd/e3v2/proui/dwin.cpp [hunk 1]
```diff
@@ -1715,6 +1715,22 @@ void dwinPrintFinished() {
...
+  #endif
+  if (all_axes_homed()) {
+    const int16_t zpos = current_position.z + TERN(NOZZLE_PARK_FEATURE,
+    NOZZLE_PARK_Z_RAISE_MIN, Z_POST_CLEARANCE);
+    _MIN(zpos, Z_MAX_POS);
+    MString<25> cmd;
+    cmd.setf(cmd, F("G0Z%i\nG0F2000Y%i"), zpos, TERN(NOZZLE_PARK_FEATURE, park_pos.y, 200));
+    queue.inject(&cmd);
+  }
+  #ifdef SD_FINISHED_RELEASECOMMAND
```
Comment (thisiskeithb) [inline Marlin/src/lcd/e3v2/proui/dwin.cpp]: `EVENT_GCODE_SD_ABORT` should be used here instead.

Comment (classicrocker883) [reply Marlin/src/lcd/e3v2/proui/dwin.cpp]: I agree

Comment (classicrocker883) [reply Marlin/src/lcd/e3v2/proui/dwin.cpp]: however, in MarlinCore.cpp, in the `loop()` function... when the abort flag is called... `if (card.flag.abort_sd_printing) abortSDPrinting();`

```y
abortSDPrinting() {...
    #ifdef EVENT_GCODE_SD_ABORT
      queue.inject(F(EVENT_GCODE_SD_ABORT));
    #endif

}
```

so this happens when for this is true 
```y
void abortFilePrintSoon() { flag.abort_sd_printing = isFileOpen(); }
```
and  
```y
bool isFileOpen()       { return isMounted() && file.isOpen(); }

/*file.isOpen() Returns:
True if this is an open file/directory else false.
*/
```

so it seems that `EVENT_GCODE_SD_ABORT` runs anyway.

Comment (thinkyhead) [reply Marlin/src/lcd/e3v2/proui/dwin.cpp]: In fact, this should be removed altogether. The `SD_FINISHED_RELEASECOMMAND` G-code is properly invoked from within `M1001` as part of SD print completion in the proper circumstances, allowing for a wider variety of G-codes to be used safely as part of `SD_FINISHED_RELEASECOMMAND`.

**Comment by thinkyhead** [comment]
I tweaked the `dwinPrintAborted` code to just use `G27` when it's available. That may also park X, but that will depend on user preference settings, which is as it should be.

Note that any `G0`/`G1` commands that include `F` will modify the feedrate for subsequent moves. If that is not wanted, the feedrate should be restored to its previous value after the `G0`/`G1` command. That might be the distinct `G0` feedrate if the option for a distinct `G0` feedrate is enabled. Another option is just to call a "blocking move" function directly. This is safe when you know there will be no more commands left in the G-code buffer that could undo the move.

**Comment by classicrocker883** [comment]
I did a pull from the current bugfix without realizing it may have made some unintended changes. appears to be okay 🤞

File: Marlin/src/sd/usb_flashdrive/lib-uhs2/Usb.cpp [hunk 2]
```diff
@@ -202,7 +202,7 @@ uint8_t USB::ctrlReq(uint8_t addr, uint8_t ep, uint8_t bmReqType, uint8_t bReque
  * rcode 0 if no errors. rcode 01-0f is relayed from dispatchPkt(). Rcode f0 means RCVDAVIRQ error, fe = USB xfer timeout
  */
 uint8_t USB::inTransfer(uint8_t addr, uint8_t ep, uint16_t *nbytesptr, uint8_t *data, uint8_t bInterval /*= 0*/) {
-  EpInfo *pep=nullptr;
+  EpInfo *pep = nullptr;
```
Comment (classicrocker883) [inline Marlin/src/sd/usb_flashdrive/lib-uhs2/Usb.cpp]: good catch, dont know how I mistaken that

**Pull request merged**